website/docs/r/vpc_peering_options.html.markdown

```diff
@@ -0,0 +1,166 @@
+---
+layout: "aws"
+page_title: "AWS: aws_vpc_peering_connection_options"
+sidebar_current: "docs-aws-resource-vpc-peering-options"
+description: |-
+  Provides a resource to manage VPC peering connection options.
+---
+
+# aws_vpc_peering_connection_options
+
+Provides a resource to manage VPC peering connection options.
+
+~> **NOTE on VPC Peering Connections and VPC Peering Connection Options:** Terraform provides
+both a standalone VPC Peering Connection Options and a [VPC Peering Connection](vpc_peering.html)
+resource with `accepter` and `requester` attributes. Do not manage options for the same VPC peering
+connection in both a VPC Peering Connection resource and a VPC Peering Connection Options resource.
+Doing so will cause a conflict of options and will overwrite the options.
+
+Basic usage:
+
+```hcl
+resource "aws_vpc" "foo" {
+  cidr_block = "10.0.0.0/16"
+}
+
+resource "aws_vpc" "bar" {
+  cidr_block = "10.1.0.0/16"
+}
+
+resource "aws_vpc_peering_connection" "foo" {
+  vpc_id        = "${aws_vpc.foo.id}"
+  peer_vpc_id   = "${aws_vpc.bar.id}"
+  auto_accept   = true
+}
+
+resource "aws_vpc_peering_connection_options" "foo" {
+  vpc_peering_connection_id = "${aws_vpc_peering_connection.foo.id}"
+
+  accepter {
+    allow_remote_vpc_dns_resolution = true
+  }
+
+  requester {
+    allow_vpc_to_remote_classic_link = true
+    allow_classic_link_to_remote_vpc = true
+  }
+}
+```
+
+Basic cross-account usage:
+
+```hcl
+provider "aws" {
+  # Requester's credentials.
+}
+
+provider "aws" {
+  alias = "peer"
+
+  # Accepter's credentials.
+}
+
+resource "aws_vpc" "main" {
+  cidr_block = "10.0.0.0/16"
+
+  enable_dns_support   = true
+  enable_dns_hostnames = true
+}
+
+resource "aws_vpc" "peer" {
+  provider   = "aws.peer"
+  cidr_block = "10.1.0.0/16"
+
+  enable_dns_support   = true
+  enable_dns_hostnames = true
+}
+
+data "aws_caller_identity" "peer" {
+  provider = "aws.peer"
+}
+
+# Requester's side of the connection.
+resource "aws_vpc_peering_connection" "peer" {
+  vpc_id        = "${aws_vpc.main.id}"
+  peer_vpc_id   = "${aws_vpc.peer.id}"
+  peer_owner_id = "${data.aws_caller_identity.peer.account_id}"
+  auto_accept   = false
+
+  tags {
+    Side = "Requester"
+  }
+}
+
+# Accepter's side of the connection.
+resource "aws_vpc_peering_connection_accepter" "peer" {
+  provider                  = "aws.peer"
+  vpc_peering_connection_id = "${aws_vpc_peering_connection.peer.id}"
+  auto_accept               = true
+
+  tags {
+    Side = "Accepter"
+  }
+}
+
+# As options can't be set until the connection has been accepted
+# create an explicit dependency on the accepter.
+
+resource "aws_vpc_peering_connection_options" "requester" {
+  vpc_peering_connection_id = "${aws_vpc_peering_connection_accepter.peer.id}"
```

Comment:
I believe this should be `"${aws_vpc_peering_connection.peer.id}"`

Reply:
Actually it should be `aws_vpc_peering_connection_accepter.peer.id`.
The comment above

> As options can't be set until the connection has been accepted
> create an explicit dependency on the accepter.

is meant to explain why - We don't want to create the requester's options until the accepter has accepted.
I'll move the comment onto the line above the code.